js/main.js

```diff
@@ -0,0 +1,25 @@
+const $photoURL = document.querySelector('#url');
+const $form = document.querySelector('form');
+
+$photoURL.addEventListener('input', event => {
+  const url = document.querySelector('#url').value;
+  document.querySelector('#img-preview').setAttribute('src', url);
```

Comment:
Same note as above ^. Utilize a variable that stores this `img` element globally.